docs/guides/0x-animate-your-visualizations.md

```diff
@@ -0,0 +1,391 @@
+
+
+```js
+const viz = new carto.Viz(`
+    filter: animation($date_time, 10, fade(1,1))
+`);
+```
+
+<div class="example-map">
+    <iframe
+        id="animation-step-0"
+        src="/developers/carto-vl/examples/maps/guides/animation/step-0.html"
+        width="100%"
+        height="500"a
+        frameBorder="0">
+    </iframe>
+</div>
+
+#### The filter property
+
+
+Now that we have a basic map of animated points, let’s get a deeper understanding of the different pieces that make up the animation expression to fine-tune the visualization.
+
+In the example above, the animation expression is set to the `filter` property (for more information see the[vizSpec](https://carto.com/developers/carto-vl/reference/#vizspec)). The convention in CARTO VL is that `0` represents the boolean value `false`, (the absence of or an *off* state), while `1` represents `true` (the presence of or an *on* state). In the context of an  animation expression assigned to a `filter`, any feature that has a  value of `0` will be “filtered-out” and not shown while any feature that meets the filter, is assigned a value of `1` and is shown. 
+
+You can think of the  animation expression like a clock, cycling through each record of the input.  Any time there is a match, ,  the feature is drawn.  Now that we’ve gone through a more in-depth discussion of how features draw, let’s take a look at the next set of animation parameters are used to define the property, speed, and transition between animated features.
+
+#### Input
+
+The first animation parameter (input), is the attribute that you want to animate. By default, the animation progresses from the attribute minimum to maximum values.
+#### Duration
+
+This parameter defines the duration of the animation progress in seconds. As stated above, during the animation cycle, all possible input values are evaluated. When an input value is matched, animation returns a value of `1` for the feature and it is drawn.
+
+#### Fade
+
+The `fade` parameter is used to define two additional (`fadeIn, fadeOut`) durations in seconds. These parameters allow for smooth transitions between features during an animation's duration. During the *`fadeIn`* phase, all features with a match will fade-in to the animation, transitioning from `0` (invisible) to `1` (visible). During the *`fadeOut`* phase features will transition from `1` back to `0` and the next set of features will begin to fade-in.
+
+### Style and configure the animation
+
+To illustrate these concepts, let's animate the journey of three birds from January to April of 2014 using data from [movebank.org](https://www.movebank.org/).
+
+#### Animate points
+
+At the beginning of the guide we created a simple point animation using the attribute `$date_time` for the temporal input (the date of the bird track and associated timestamp), set the duration to `10` seconds and both fade parameters to `1`. 
```

Comment:
I feel like this and the first "Create a basic animation" should be revised (maybe even merged somehow), and include more details (and code snippets) for the user to follow the guide in a "hands-on mode"